src/ai/backend/storage/filebrowser/filebrowser.py

```diff
@@ -0,0 +1,208 @@
+    storage_proxy_root_path_index = p.parts.index("storage")
+    settings_path = Path(*p.parts[0 : storage_proxy_root_path_index + 1]) / "filebrowser_dir/"
+    _, requested_volume = host.split(":")
+    volumes = ctx.local_config["volume"]
+    for volume_name in volumes.keys():
+        if requested_volume == volume_name:
+            volume_cls: Type[AbstractVolume] = BACKENDS[volumes.get(volume_name)["backend"]]
+            mount_path = Path(volumes.get(volume_name)["path"])
+            volume_obj = volume_cls(
+                local_config=ctx.local_config,
+                mount_path=mount_path,
+                fsprefix=None,
+                options={},
+            )
+    port_range = ctx.local_config["filebrowser"]["port_range"].split("-")
+    service_port = get_available_port(port_range)
+    running_docker_containers = await get_filebrowsers()
+    if len(running_docker_containers) >= max_containers:
+        print(
+            "Can't create new container. Number of containers exceed the maximum limit.",
+        )
+        return ("0", 0, "0")
+    await prepare_filebrowser_app_config(settings_path, service_port)
+    async with closing_async(aiodocker.Docker()) as docker:
+        config = {
+            "Cmd": [
+                "/filebrowser_dir/start.sh",
+                f"{user_id}",
+                f"{group_id}",
+                f"{service_port}",
+            ],
+            "ExposedPorts": {
+                f"{service_port}/tcp": {},
+            },
+            "Image": image,
+            "HostConfig": {
+                "PortBindings": {
+                    f"{service_port}/tcp": [
+                        {
+                            "HostIp": f"{service_ip}",
+                            "HostPort": f"{service_port}/tcp",
+                        },
+                    ],
+                },
+                "CpuCount": cpu_count,
+                "Memory": memory,
+                "Mounts": [
+                    {
+                        "Target": "/filebrowser_dir/",
+                        "Source": f"{settings_path}",
+                        "Type": "bind",
+                    },
+                ],
+            },
+        }
+        for vfolder in vfolders:
+            filebrowser_mount_path = str(
+                volume_obj.mangle_vfpath(UUID(vfolder["vfid"])),
+            )
+            config["HostConfig"]["Mounts"].append(
+                {
+                    "Target": f"/data/{str(vfolder['name'])}",
+                    "Source": filebrowser_mount_path,
+                    "Type": "bind",
+                },
+            )
+        container_name = f"ai.backend.container-filebrowser-{service_port}"
+        container = await docker.containers.create_or_replace(
+            config=config,
+            name=container_name,
+        )
+        container_id = container._id
+        await container.start()
+    tracker_db = FilebrowserTrackerDB(db_path)
```

Comment:
`FilebrowserTrackerDB`'s initializer calls some blocking calls (related to sqlite) and this does isn't good for asyncio codes. We need to split out blocking I/O calls onto separate function (maybe `__ainit__()`?).